.github/workflows/examples-openiotsdk.yaml

```diff
@@ -46,10 +46,11 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
+                testItem: ["test0", "test1", "test2", "test3", "test4", "test5"]
                 socketAPI: ["iotsocket", "lwip"]
                 
-        name: Open IoT SDK examples building
-        timeout-minutes: 140
+        name: Open IoT SDK examples building and testing
+        #timeout-minutes: 140
```

Comment:
140 minutes is not enough ? 
Why not raise the timeout ?

Reply:
I commented on it for testing purposes. Need to be restored. However, the timeouts need to be adjusted, according to the test results. I'll change them.